grammar.js

```diff
@@ -440,6 +441,13 @@ module.exports = grammar({
     _case_clause_tuple_access: ($) =>
       seq(field("tuple", $.identifier), ".", field("index", $.integer)),
     let: ($) => seq("let", $._assignment),
+    use: ($) =>
+      seq(
+        "use",
+        series_of($.identifier, ","),
```

Comment:
I believe this should be 
```suggestion
        optional(series_of($.identifier, ",")),
```
as having no bindings is also valid, e.g.

```gleam
use <- do_thing()
```